packages/react-workers/__tests__/sorting.test.js

```diff
@@ -0,0 +1,79 @@
+import { applySorting } from '../src/utils/sorting';
+
+const features = [
+  { column1: 'C', column2: 3 },
+  { column1: 'A', column2: 1 },
+  { column1: 'B', column2: 2 },
+  { column1: 'D', column2: 4 },
+  { column1: 'D', column2: 5 }
+];
+
+const commonSortedFeatures = [
+  { column1: 'A', column2: 1 },
+  { column1: 'B', column2: 2 },
+  { column1: 'C', column2: 3 },
+  { column1: 'D', column2: 4 },
+  { column1: 'D', column2: 5 }
+];
+
+const commonSortedFeatures2 = [
+  { column1: 'D', column2: 4 },
+  { column1: 'D', column2: 5 },
+  { column1: 'C', column2: 3 },
+  { column1: 'B', column2: 2 },
+  { column1: 'A', column2: 1 }
+];
+
+describe('Sorting', () => {
+  describe('should correctly understand sortOptions', () => {
+    test('if undefined', async () => {
+      expect(applySorting(features)).toEqual(features);
+    });
+
+    test('if sortBy is string', async () => {
+      expect(applySorting(features, { sortBy: 'column1' })).toEqual(commonSortedFeatures);
+    });
+
+    test('if sortBy is array of string', async () => {
```

Comment:
I guess this is just using both columns, with default 'asc' order. If that's the case, then I would probably suggest to rename the test to something like...

```suggestion
    test('if sortBy uses 2 columns', async () => {
```

Reply:
Fine!